sources/ClangSharp.PInvokeGenerator/PInvokeGenerator.VisitDecl.cs

```diff
@@ -1590,6 +1595,11 @@ void VisitAnonymousRecordDecl(RecordDecl recordDecl, RecordDecl nestedRecordDecl
 
             void VisitAnonymousRecordDeclFields(RecordDecl rootRecordDecl, RecordDecl anonymousRecordDecl, string contextType, string contextName)
             {
+                if (_config.ExcludeAnonymousFieldHelpers)
```

Comment:
Do you think this should be a general `ExcludeHelpers` or that there should be an `Exclude*Helper` per "helper type" (just to cover any future helper additions)